scraper.py

```diff
@@ -5,31 +5,39 @@
 import csv
 import sys
 import mysql.connector
+from plyer import notification
 
-c = sys.argv[1]
-yr = sys.argv[2]
+c = sys.argv[0]
+yr = sys.argv[0]
 if (c == "ec"):
     st = "PES2"
 else:
     st = "PES1"
 
 
+notification.notify(
```

Comment:
This has already been solved in another issue. Your issue wasn't about this. Your issue was to make a telegram bot